plenum/test/consensus/order_service/test_sim_order_during_view_change.py

```diff
@@ -12,28 +12,17 @@
 REQUEST_COUNT = 10
 
 
-@pytest.mark.parametrize("seed", range(100))
-def test_view_change_while_ordering_with_real_msgs_default_seed(seed):
-    do_test(seed)
+def test_view_change_while_ordering_with_real_msgs_random_seed(random):
+    do_test(random)
 
 
```

Comment:
Why did we remove fixed seed here? I think we need to keep for regression test purposes.

Reply:
I removed only simple seed from 0 to 100.

Reply:
I returned back fixed seed.